<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataEnvelopeUI</class>
 <widget class="QWidget" name="DataEnvelopeUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <item>
     <widget class="QCheckBox" name="checkBoxEnableEnvelope">
      <property name="text">
       <string>Enable envelope</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetAmplitude" native="true">
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="hlayoutAmplitude">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetTime" native="true">
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="hlayoutTime">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
